src/System.Threading.Timer/src/Configurations.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      net463|AnyOS;
```

Comment:
It feels to me that this should be an ItemGroup with additional metadata - what's the rationale behind the semicolon separated list with metadata split with pipe?